CHANGES.rst

```diff
@@ -26,6 +26,8 @@ Enhancements and Fixes
   any model serialized in VO-DML. This package dynamically generates python objects
   whose structure corresponds to the classes of the mapped models. [#497]
 
+- MIVOT module: the model references in the dictionaries that are used to build ``MivotInstance``
+  objects are made more consistent [#551]
```

Comment:
Don't worry about changing anything for this now, I would prefer to have the changelog check in CI to fail now to remind me at release time to consolidate all the related ones into something that is most useful for the users.